app/models/contributor.rb

```diff
@@ -0,0 +1,20 @@
+class Contributor < ApplicationRecord
+  WORKLOADS = [20, 25, 30, 40].freeze
+  enum contributor_type: { contributor: 0, admin: 1 }
```

Comment:
I'm not sure about having admin contributors. I'd prefer to have a User model that will manage the application and the Contributor model that has information of each contributor. But it depends on the requirements.

Reply:
I agree 👍

Reply:
After carefully reviewing the project requirements, we don't really need to have a User model, we are just fine with the Contributors with the admin privilidge. Although I would change the name of the enum keys for something like:
```suggestion
  enum contributor_type: { standard: 0, admin: 1 }
```

Reply:
Acording to this card https://trello.com/c/Oe8nup1G/7-crear-modelo-de-roles
We should have an association with the roles, so lets remove this enum

I am so sorry of this pulling and throwing, I will see a way to avoid this confusion in the future